apps/gendsa.c

```diff
@@ -117,6 +119,12 @@ int gendsa_main(int argc, char **argv)
         goto end2;
 
     DSA_get0_pqg(dsa, &p, NULL, NULL);
+
+    if (BN_num_bits(p) > DSA_RECOMMENDED_MAX_KEY_LENGTH)
+       BIO_printf(bio_err, "# Warning: It is not recommended to use more than %d bit for DSA keys.       #\n"
```

Comment:
Indentation: Insert linebreak before argument 2 (the literal string) and align the two literal strings with the `b` of `bio_err`.